include/arch/arm/aarch64/arch.h

```diff
@@ -29,7 +29,9 @@
 #include <arch/arm/aarch64/sys_io.h>
 #include <arch/arm/aarch64/timer.h>
 #include <arch/arm/aarch64/error.h>
+#ifdef CONFIG_ARM_MMU
```

Comment:
Instead of the ifdef on `ARM_MMU` it's probably better to differentiate on the cortex type just in case in the future we have other includes that are going to be specific for the Cortex-R82.

Reply:
Ah, according to https://www.arm.com/products/silicon-ip-cpu/cortex-r/cortex-r82, the Cortex-R82 could support MMU depends on hardware design. Some of the Cortex-R82 support MMU and some support MPU. So maybe we cannot simply differentiate on the cortex type, in case we want to enable MMU on Cortex-R82?  May I ask your suggestions?

Reply:
Ok, in that case having `CONFIG_ARM_MMU` here is fine. The board / soc defconfig will specify with `CONFIG_ARM_MMU` or `CONFIG_ARM_MPU` if the soc is supporting either MMU or MPU.